FirebaseFacebookAuthUI/Sources/FUIFacebookAuth.m

```diff
@@ -322,14 +320,14 @@ - (void)completeSignInFlowWithAccessToken:(nullable NSString *)accessToken
   }
   FIRAuthCredential *credential;
   if (idToken) {
-    NSString *rawNonce = self->_currentNonce;
+    NSString *rawNonce = self.currentNonce;
```

Comment:
is a nil check necessary here?

Reply:
I think if this happens to be `nil`, we can just propagate the backend error instead of throwing an exception here.